From f3dcee506df03cc47ca0e3312c25020b5f9b0105 Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Mon, 25 Jan 2021 14:51:14 +0800
Subject: [PATCH 1/2] disable glx when build xwayland glamor

fix Xwayland error:
/usr/bin/Xwayland: /lib/aarch64-linux-gnu/libGL.so.1: no version information available
3061:_ConnectPixmap reusing surface

Signed-off-by: Xianzhong <xianzhong.li@nxp.com>
---
 hw/xwayland/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hw/xwayland/meson.build b/hw/xwayland/meson.build
index 36bf213..5ba9404 100644
--- a/hw/xwayland/meson.build
+++ b/hw/xwayland/meson.build
@@ -96,8 +96,6 @@ executable(
         libxserver_xext_vidmode,
         libxserver_xkb_stubs,
         libxserver_xi_stubs,
-        libxserver_glx,
-        libglxvnd,
     ],
     install: true,
 )
-- 
2.17.1

